### DISCUSSION TITLE
easeir navigation between segments—UI suggestion

### DISCUSSION THREAD

**Discussion by yotamguttman** [mbnuqw/sidebery#1502] [Ideas]
the current interface of switching between different segments (tabs, bookmarks, history) in Sidebery is via a menu. I'm simply too lazy and rather avoid the extra clicks to change the views, even though it's very useful to have access to my bookmarks through Sidebery's sidebar...
![current](https://github.com/mbnuqw/sidebery/assets/92977002/8779cd2e-6ddf-4cae-b4e9-65ef64a32066)

**I couldn't find anything similar in the previous submissions so I'd like to suggest to turn it into 'segment controls interface'. the user can choose if they wish to have labels (like in the menu) or not:**
![unlabeled](https://github.com/mbnuqw/sidebery/assets/92977002/c2a5454e-ce12-41fc-86fd-5f8ea26a4379)
![labeled](https://github.com/mbnuqw/sidebery/assets/92977002/31a0b2b2-86f9-4dec-a5e0-204a51124be2)
this way you significantly reduce the amount of clicks needed to change segments and can quickly jump to whatever view that's relevant at the moment. it makes the sidebar useful for other things other than the tabs :)

sidenote: currently Sidebery uses a salad of its own icons along with Firefox's ones. it makes it look a little chaotic. I changed the iconography to Firefox's official ones, [all SVGs are available on Githab](https://github.com/FirefoxUX/acorn-icons/tree/main) :)

**Comment by mbnuqw** [comment]
This is also not part of Sidebery. It's Firefox's [sidebar header](https://support.mozilla.org/en-US/kb/use-firefox-sidebar-access-bookmarks-history-synced).
Sidebery's (and any other sidebar addon) area (red):
![screenshot-2024-03-04 15:50:24](https://github.com/mbnuqw/sidebery/assets/6276694/bbc6790f-d14b-4d37-bee3-2877d47b18f5)

Anything outside that area cannot be controlled by Sidebery (except sidebery pages and popups).

btw. Your concept is awesome, you could suggest it to https://connect.mozilla.org/ or maybe even as a feature request in https://bugzilla.mozilla.org/

**Reply by yotamguttman** [reply]
thanks for the explanation :)
